### ISSUE TITLE
Multi-step donation form UI

### ISSUE THREAD

**Issue by ericnicolaas** [Charitable/Davy-Donation-Form-Block#2]
@wesfranklin Can I get you to work on a design for the Davy donation form? 

Ideally, the way I'd see this working is that it would look pretty similar to what you have already designed for Charitable's refresh:

https://github.com/Charitable/Charitable/issues/800

But the Davy donation form is structured a little differently, with a multi-step donation form and less fields. 

At the moment, this is what I have:

## Amount section

![Screenshot 2020-07-20 16 19 59](https://user-images.githubusercontent.com/676582/87908014-e3971c80-caa4-11ea-97f0-ff71a7eb785a.png)

## Details section with payment buttons

![Screenshot 2020-07-20 16 20 05](https://user-images.githubusercontent.com/676582/87908034-eb56c100-caa4-11ea-913a-78ebe05cf149.png)

**Comment by ericnicolaas** [comment]
Note: the payment buttons at the bottom are not customizable — that's PayPal's default.

**Comment by wesfranklin** [comment]
@ericnicolaas Am I correct to assume that the 'Amount' and 'Details' aren't clickable? If so, they should probably be styled differently than the things that are clickable.
![Screen Shot 2020-07-21 at 11 12 14 am](https://user-images.githubusercontent.com/13580400/88001334-450dc880-cb43-11ea-82c3-57afa531d9ac.png)
![Screen Shot 2020-07-21 at 11 12 32 am](https://user-images.githubusercontent.com/13580400/88001335-46d78c00-cb43-11ea-99ca-d7112ff49857.png)

**Comment by ericnicolaas** [comment]
@wesfranklin Yeah, they wouldn't be clickable. I wonder if it would work better if there was no margin between them, so they're part of one continuous block.

**Comment by wesfranklin** [comment]
@ericnicolaas Yep, that'd work.
